mtp_api/apps/mtp_auth/models.py

```diff
@@ -21,7 +32,16 @@ class ApplicationUserMapping(TimeStampedModel):
     application = models.ForeignKey('oauth2_provider.Application')
 
     def __str__(self):
-        return self.user.username
+        return '%s -> %s' % (self.user.username, self.application.client_id)
+
+
+class ApplicationGroupMapping(TimeStampedModel):
```

Comment:
i wish we did, even if they don't load automatically it's useful as documentation cos all the pieces are hard to find/remember